src/main/java/net/openhft/chronicle/core/time/PosixTimeProvider.java

```diff
@@ -23,6 +23,8 @@
 
 /**
  * This is the raw Posix time, if you need it to be unique or a multiple of 32 use a wrapper.
+ * <p>
+ * PosixTimeProvider calls directly into clock_gettime to get the nanosecond wall clock on platforms which support it.
```

Comment:
maybe add "(native code)" after clock_gettime to make it clear that this calls into native code